error.c:25:1: warning: implicitly declaring library function 'exit' with type 'void (int) __attribute__((noreturn))' [-Wimplicit-function-declaration]
exit (status);
^
error.c:25:1: note: include the header <stdlib.h> or explicitly provide a declaration for 'exit'
--- src/error.c.orig	2001-05-05 07:10:24.000000000 -0500
+++ src/error.c	2018-11-05 07:16:16.000000000 -0600
@@ -20,6 +20,8 @@
 
 */
 
+#include <stdlib.h>
+
 void error(int status, int errnum, ...)
 {
 exit (status);
